aptos/token_bridge/sources/bridge_structs.move

```diff
@@ -26,6 +26,26 @@ module token_bridge::bridge_structs {
         // Amount of tokens (big-endian uint256) that the user is willing to pay as relayer fee. Must be <= Amount.
         fee: U256, //should be u256
     }
+    public fun create_transfer(
```

Comment:
should be `public(friend)` - we should protect/hide our struct constructors as a general principle

Reply:
done